src/js/components/CheckBoxGroup/__tests__/CheckBoxGroup-test.js

```diff
@@ -64,8 +64,12 @@ describe('CheckBoxGroup', () => {
     const component = renderer.create(
       <Grommet>
         <CheckBoxGroup disabled options={['First', 'Second']} />
-        <CheckBoxGroup options={[{ label: 'First', disabled: true }]} />
-        <CheckBoxGroup options={[{ label: 'First', disabled: true }]} />
+        <CheckBoxGroup
+          options={[{ label: 'First', value: 'First', disabled: true }]}
```

Comment:
If you change the test, you need to update the snapshots `yarn test-update`.
And according to the issue, this PR is trying to solve, I think you want to add a new test with similar labels on the same CheckBoxGroup, to verify that the keys are different.

Reply:
Key aren't rendered in snapshots and I didn't find a correct way to test the duplication. But It's a simple concatenation of two variables

Reply:
I think we shouldn't change this test since you added a new test below to cover your use case. Let's keep this test intact.